### PR TITLE
Various small fixes and improvements

### PR DESCRIPTION
**Pull request by Rafficer**
* Moving logger imports under the correct comment
* Flatten logging of arguments for better log file readability
* Move docopt parsing below initial logging to see what the user typed in the command line in case the syntax is wrong
* Improve Version printout (`2.x.x` -> `ProtonVPN-CLI v2.x.x`)
* Remove unnecessary printout that I used for debugging and forgot to remove
* Remove useless variable definition that shouldn't be in there to begin with

### CODE REVIEW

File: protonvpn_cli/utils.py [hunk 1]
```diff
@@ -11,8 +11,8 @@
 import random
 # External Libraries
 import requests
-from .logger import logger
 # Constants
+from .logger import logger
```
Comment (kaplun) [inline protonvpn_cli/utils.py]: Strictly speaking that is not a constant

File: protonvpn_cli/cli.py [hunk 2]
```diff
@@ -77,18 +77,19 @@ def main():
...
     change_file_owner(os.path.join(CONFIG_DIR, "pvpn-cli.log"))
     logger.debug("###########################")
     logger.debug("### NEW PROCESS STARTED ###")
     logger.debug("###########################")
     logger.debug(sys.argv)
-    logger.debug("Arguments\n{0}".format(args))
     logger.debug("USER: {0}".format(USER))
     logger.debug("CONFIG_DIR: {0}".format(CONFIG_DIR))
 
+    args = docopt(__doc__, version="ProtonVPN-CLI v{0}".format(VERSION))
```
Comment (kaplun) [inline protonvpn_cli/cli.py]: This overrides previous commit so you could squash them.

**Comment by kaplun** [comment]
Just small nits :)

File: protonvpn_cli/utils.py (hunk 1 above)
Comment (Rafficer) [reply protonvpn_cli/utils.py]: Looks like I wasn't paying attention...

**Comment by Rafficer** [comment]
Thanks, Should be resolved now :)

**Comment by kaplun** [comment]
Just small comment for the future: the 1st commit mention "Flatten" referring to the fact that this is replacing the previous list of commits. This was more a message to me, rather than to someone who is accessing the source code history.

Also Try to stick with commit log respecting the 50/72 (subject/body) width :-)

**Pull request merged**